src/oncall/user_sync/ldap_sync.py

```diff
@@ -138,6 +138,7 @@ def fetch_ldap():
                 name = name.decode("utf-8")
             mobile = ldap_dict.get(LDAP_SETTINGS['attrs']['mobile'])
             mail = ldap_dict.get(LDAP_SETTINGS['attrs']['mail'])
+            pushover = ldap_dict.get(LDAP_SETTINGS['attrs']['pushover'])
```

Comment:
this looks like it could potentially cause keyerror exceptions if there is no pushover data in ldap. The app should look and behave as it does now if there is no pushover data defined anywhere in ldap, iris, or oncall.